crates/compiler/test_gen/src/gen_list.rs

```diff
@@ -3520,6 +3516,16 @@ fn list_walk_backwards_implements_position() {
     );
 }
 
+#[test]
+#[cfg(any(feature = "gen-llvm", feature = "gen-wasm"))]
+fn list_walk_backwards_until_sum() {
+    assert_evals_to!(
+        r#"List.walkBackwardsUntil [1, 2] 0 \a,b -> Continue (a + b)"#,
+        3,
+        i64
+    );
+}
+
```

Comment:
Not a big deal, but is there a reason to reorder the test here?

Reply:
One of the other merges separated this test from another test of the same func, so I just moved them one below another